data/interfaces/default/library.html

```diff
@@ -590,24 +602,38 @@ <h4 class="modal-title" id="myModalLabel">Confirm Delete</h4>
 
         $('#transcode_decision-selection').on('change', function () {
             $('#transcode_decision-selection > label').removeClass('active');
-            var selected_filter = $('input[name=transcode_decision-filter]:checked', '#transcode_decision-selection');
+            let selected_filter = $('input[name=transcode_decision-filter]:checked', '#transcode_decision-selection');
             $(selected_filter).closest('label').addClass('active');
             transcode_decision = $(selected_filter).map(function () { return $(this).val(); }).get().join(',');
             setLocalStorage('library_' + section_id + 'history_transcode_decision', transcode_decision);
             history_table.draw();
         });
+
+        $('#network_type-selection').on('change', function () {
+            $('#network_type-selection > label').removeClass('active');
+            let selected_filter = $('input[name=network_type-filter]:checked', '#network_type-selection');
+            $(selected_filter).closest('label').addClass('active');
+            network_type = $(selected_filter).map(function () { return $(this).val(); }).get().join(',');
+            setLocalStorage('library_' + section_id + 'history_network_type', network_type);
+            history_table.draw();
+        });
     }
 
     $('#nav-tabs-history').on('shown.bs.tab', function() {
         if (typeof(history_table) === 'undefined') {
-            var transcode_decision = getLocalStorage('library_' + section_id + 'history_transcode_decision', 'all');
+            const transcode_decision = getLocalStorage('library_' + section_id + 'history_transcode_decision', 'all');
```

Comment:
```suggestion
            const transcode_decision = getLocalStorage('library_' + section_id + '-history_transcode_decision', 'all');
```

Also fix `setLocalStoage` on line 608.